README.md

```diff
@@ -78,37 +78,56 @@ As RequestKit was designed to handle OAuth requests we needed something to store
 ```swift
 public struct TokenConfiguration: Configuration {
     public let accessToken: String?
-    public let apiEndpoint = "https://my.webservice.example/api/2.0/"
+    public let apiEndpoint = "https://my.webservice.example/api/2.0/"    
     public let accessTokenFieldName = "access_token"
     public let errorDomain: String = "com.my.customErrorDomain"
+    
+    public init(_ accessToken: String? = nil) {
+        self.accessToken = accessToken
+    }
+}
+```
+
+In the above `Configuration` the `accessToken` will be passed as a URL parameter named `access_token` with each request. Alternatively you can have the `accessToken` passed in an HTTP Authorization header by setting the `authorizationHeader` property to the desired token type. As an example the following `Configuration` passes it as a Bearer token.
 
+```swift
+public struct TokenConfiguration: Configuration {
+    public let accessToken: String?
+    public let apiEndpoint = "https://my.webservice.example/api/2.0/"
+    public let authorizationHeader: String? = "Bearer"
```

Comment:
We can omit the `: String?` here.

Reply:
Frustratingly it appears that we can't omit the type here. I'm not sure if this is the expected Swift behavior or if I'm doing something dumb or if this is a Swift bug? I put together some code you can throw into a playground to demo the issue:
https://gist.github.com/mpvosseller/02c6bbf853442c5de3c2fc9adc139e00